docs/user-guide/advanced.md

```diff
@@ -4,6 +4,29 @@ title: User Guide (advanced)
 sidebar_label: Advanced options
 ---
 
-Welcome to the user guide!
+There are some options to tweak the invitation link to unlock more features in
+Jitsi. The following parameters apply to the web version only.
+They can be combined using `&`, e.g. `#config.defaultLanguage=en&config.minParticipants=3`.
 
-Check back soon!
+<!--
+  See also for implementation:
+  https://github.com/jitsi/jitsi-meet/blob/b0188a71841c966122c3cce8c7023b7de8e32a82/config.js
+  https://github.com/jitsi/jitsi-meet/blob/b0188a71841c966122c3cce8c7023b7de8e32a82/react/features/base/config/configWhitelist.js
+  https://github.com/jitsi/jitsi-meet/blob/b0188a71841c966122c3cce8c7023b7de8e32a82/react/features/base/config/functions.any.js#L70
+-->
+
+* `#config.defaultLanguage=en` will change the UI default language
```

Comment:
Any chance you can perhaps group the options semantically? Like audio, video, privacy, etc.

Reply:
I was first going through the lists (I left as HTML comment) to see, whether the PR would have a chance at all.
Adding groups with headlines now.